inst/old_function/vs_lrstat.R

```diff
@@ -0,0 +1,89 @@
+library(dplyr)
+library(gsDesign2)

```

Comment:
The folder should be removed later. If we need this file, please move it to another location.